test/websocket/stream.cpp

```diff
@@ -477,59 +491,8 @@ class stream_test
         }
     }
 
-    struct con
-    {
-        stream<socket_type> ws;
-
-        con(endpoint_type const& ep, boost::asio::io_service& ios)
-            : ws(ios)
-        {
-            ws.next_layer().connect(ep);
-            ws.handshake("localhost", "/");
-        }
-    };
-
-    template<std::size_t N>
-    class cbuf_helper
-    {
-        std::array<std::uint8_t, N> v_;
-        boost::asio::const_buffer cb_;
-
-    public:
-        using value_type = decltype(cb_);
-        using const_iterator = value_type const*;
-
-        template<class... Vn>
-        explicit
-        cbuf_helper(Vn... vn)
-            : v_({{ static_cast<std::uint8_t>(vn)... }})
-            , cb_(v_.data(), v_.size())
-        {
-        }
-
-        const_iterator
-        begin() const
-        {
-            return &cb_;
-        }
-
-        const_iterator
-        end() const
-        {
-            return begin()+1;
-        }
-    };
-
-    template<class... Vn>
-    cbuf_helper<sizeof...(Vn)>
-    cbuf(Vn... vn)
-    {
-        return cbuf_helper<sizeof...(Vn)>(vn...);
-    }
-
     void testClose(endpoint_type const& ep, yield_context do_yield)
```

Comment:
Its currently not called but I plan on circling back at some point and trying to get those last few lines that don't have coverage.